robot_sort/robot_sort.py

```diff
@@ -101,6 +101,8 @@ def sort(self):
         # intially has no items.
         self.set_light_on()
         self.swap_item()
+       
```

Comment:
Nice, job staying with it and getting all the tests to pass Michael 👍 All those print statements, thought terminal was in an infinite loop 😆